lib/logcat.js

```diff
@@ -13,13 +13,26 @@ class Logcat {
   }
 
   startCapture () {
-    return new Promise(async (resolve, reject) => {
-      log.debug("Starting logcat capture");
+    let started = false;
+    return new Promise(async (_resolve, _reject) => {
+      const resolve = function (...args) {
```

Comment:
It doesn't make use of anything that the fat arrow would provide. So there is no need.